pkg/kubectl/cmd/delete.go

```diff
@@ -69,10 +69,14 @@ Examples:
 				NamespaceParam(cmdNamespace).DefaultNamespace().
 				FilenameParam(flags.Filenames...).
 				SelectorParam(cmdutil.GetFlagString(cmd, "selector")).
-				ResourceTypeOrNameArgs(args...).
+				SelectedResourceTypeOrNameArgs(args...).
 				Flatten().
 				Do()
-			checkErr(r.Err())
+
+			if r.Err() != nil {
```

Comment:
Hrm - this change means that the contract of resource builder changes (if it returns an error, you *cannot* use the visited responses).  What was it for?